docker/docker-compose-with-lakekeeper.yml

```diff
@@ -0,0 +1,596 @@
+              break
+            else
+              echo "✗ Bootstrap conflict"
+              cat /tmp/bootstrap.json 2>/dev/null || true
+              exit 1
+            fi
+          else
+            echo "Bootstrap attempt $$attempt failed (HTTP $$http_code)"
+            if [ $$attempt -eq 5 ]; then
+              cat /tmp/bootstrap.json 2>/dev/null || true
+              exit 1
+            fi
+            sleep $$((2 ** attempt))
+          fi
+        done
+        
+        # Create default warehouse
+        echo "Creating default warehouse..."
+        sleep 2
+        for attempt in 1 2 3 4 5; do
+          response=$$(curl -s -w "%{http_code}" -X POST \
+            -H "Content-Type: application/json" \
+            -d "{
+              \"warehouse-name\": \"risingwave-warehouse\",
+              \"storage-profile\": {
+                \"type\": \"s3\",
+                \"bucket\": \"hummock001\",
+                \"key-prefix\": \"risingwave-lakekeeper\",
+                \"region\": \"us-east-1\",
+                \"sts-enabled\": false,
+                \"flavor\": \"s3-compat\",
+                \"endpoint\": \"http://minio-0:9301/\",
+                \"path-style-access\": true
+              },
+              \"storage-credential\": {
+                \"type\": \"s3\",
+                \"credential-type\": \"access-key\",
+                \"aws-access-key-id\": \"hummockadmin\",
+                \"aws-secret-access-key\": \"hummockadmin\"
+              }
+            }" \
+            http://lakekeeper:8181/management/v1/warehouse \
+            -o /tmp/warehouse.json)
+          
+          http_code=$${response##*[^0-9]}
+          
+          if [ "$$http_code" = "200" ] || [ "$$http_code" = "201" ]; then
+            echo "✓ Successfully created default warehouse!"
+            break
+          elif [ "$$http_code" = "400" ]; then
+            if grep -q "Storage profile overlaps" /tmp/warehouse.json 2>/dev/null; then
+              echo "✓ Warehouse already exists, skipping..."
+              break
+            else
+              echo "✗ Warehouse creation failed"
+              cat /tmp/warehouse.json 2>/dev/null || true
+              exit 1
+            fi
+          else
+            echo "Warehouse creation attempt $$attempt failed (HTTP $$http_code)"
+            if [ $$attempt -eq 5 ]; then
+              cat /tmp/warehouse.json 2>/dev/null || true
+              exit 1
+            fi
+            sleep $$((2 ** attempt))
+          fi
+        done
+        
+        echo "🎉 Lakekeeper setup completed successfully!"
+        echo "Lakekeeper is ready with warehouse: risingwave-warehouse"
+      '
+
+volumes:
+  postgres-0:
+    external: false
+  grafana-0:
+    external: false
+  minio-0:
+    external: false
+  prometheus-0:
+    external: false
+  message_queue:
+    external: false
+  lakekeeper-db:
+    external: false
+
+networks:
+  default:
+    driver: bridge
```

Comment:
Why do we need this? I don't see we enable bridge mode in other docker compose files.

Reply:
The bridge actually is the default networks type if we don't specify it. https://arc.net/l/quote/ontfwara